src/joy_api.c

```diff
@@ -109,75 +109,137 @@ static unsigned int joy_splt_format_data(flow_record_t *rec,
                                          unsigned char *data)
 {
     unsigned int i = 0;
+    unsigned int entry_counter = 0;
```

Comment:
why not use uint32_t?

Reply:
because I stayed consistent with the declaration type above it. The entry count will never exceed 4 bits anyways, so in reality a uint8_t could have been used. But I stayed consistent. It doesn't matter though.